ui/src/lib/fetch/transformers/useApi.ts

```diff
@@ -1,18 +1,97 @@
-import { GetAction, GetActions, GetActionsSegregated } from "@flow/types";
+import { useMemo } from "react";
+
+import {
+  Action,
+  GetAction,
+  GetActions,
+  GetActionsSegregated,
+} from "@flow/types";
 
 import { useFetch } from "./useFetch";
 
+const filterActions = (
```

Comment:
Having functions used inside a custom hook below the custom hook has become kind of a convention. IF you can move these below that would be awesome 🙏🏻